pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -25,6 +32,5 @@ stdenv.mkDerivation rec {
     description = "Build tool that builds code quickly and reliably";
     license = stdenv.lib.licenses.asl20;
     maintainers = [ stdenv.lib.maintainers.philandstuff ];
-    platforms = [ "x86_64-linux" ];
```

Comment:
for what it's worth, I'm fine with not using nix to install bazel on darwin.  It's very self-contained so I don't mind doing it "the normal way".